recipe/meta.yaml

```diff
@@ -10,30 +9,17 @@ package:
   version: {{ version }}
 
 source:
-  fn: spams-python2-{{ version }}.tar.gz                                                                                         # [py2k]
-  url: http://spams-devel.gforge.inria.fr/hitcounter2.php?file={{ file_num }}/spams-python-v{{ version }}-svn{{ date }}.tar.gz   # [py2k]
-  sha256: b0372195c5c6e42251caad8751907873392ee0f87dc00767d45cd1d19a8e5e21                                                       # [py2k]
-
-  fn: spams-python3-{{ version }}.tar.gz                                                                                         # [py3k]
-  url: http://spams-devel.gforge.inria.fr/hitcounter2.php?file={{ file_num }}/spams-python3-v{{ version }}-svn{{ date }}.tar.gz  # [py3k]
-  sha256: bd86be7bbe25dffffd5cc969ba5ec71e52f0a45102e31e40ebedf66616788734                                                       # [py3k]
+  fn: spams-python-{{ version }}.tar.gz
+  url: http://spams-devel.gforge.inria.fr/hitcounter2.php?file={{ file_num }}/spams-python-v{{ version }}-svn{{ date }}.tar.gz
+  sha256: 3012529c0461bd777228dce53c880f2a5fa2aaf22b5d673a1f22045a63f4d8bc
 
   patches:
-    # Fix how BLAS is detected. Mailed to developer.
-    - python2_setup.py.patch      # [py2k]
-    - python3_setup.py.patch      # [py3k]
-    # Update bindings for NumPy 1.10.x from 1.9.x.
-    - spams_wrap.cpp.patch
-    - spams_wrap-windows.cpp.patch
     # Handle type mismatch issues in array creation.
-    - python2_spams.py.patch      # [py2k]
-    - python3_spams.py.patch      # [py3k]
-    - python2_test_prox.py.patch  # [py2k]
-    - python3_test_prox.py.patch  # [py3k]
+    - spams.py.patch
```

Comment:
Can we refresh the comment above to describe this? Also did this get submitted upstream yet?

Reply:
will check.

Reply:
not in upstream yet. https://gitlab.inria.fr/thoth/spams-devel/blob/master/swig/python/spams.py